packages/vue-app/template/mixins/fetch.server.js

```diff
@@ -26,6 +26,9 @@ async function serverPrefetch() {
   attrs['data-fetch-key'] = this._fetchKey
 
   // Add to ssrContext for window.__NUXT__.fetch
+  if (process.env.NODE_ENV === 'development' && this.$ssrContext.nuxt.fetch[this._fetchKey] !== undefined) {
```

Comment:
We have options.debug (template option) to allow debugging in production/staging